lib/scrabble_scoring.rb

```diff
@@ -0,0 +1,76 @@
+module Scrabble
+  class Scoring
+    LETTERS = {
```

Comment:
Wouldn't it take as much time to calculate since in both cases, the computer will go through each letter to find the value of the given letter anyway?  In case of LETTERS[S], it will still need to go through all the letters until it finds key "s" as well.  I don't see the benefit of using this way... Although it looks easier to read, I believe.  Is it(not using each loop) considered DRYer?